test/build/50_ch-image.bats

```diff
@@ -65,6 +65,51 @@ EOF
     [[ $output != *"delete/test_stage0"* ]]
     [[ $output != *"delete/test_stage1"* ]]
     [[ $output != *"delete/test_stage2"* ]]
+
+    # Delete list of images
+    ch-image build -t tmpimg1 -f - . << 'EOF'
+FROM alpine:3.17
+EOF
+    ch-image build -t tmpimg2 -f - . << 'EOF'
+FROM alpine:3.17
+EOF
+    ch-image delete tmpimg1 tmpimg2
+    run ch-image list
+    echo "$output"
+    [[ $status -eq 0 ]]
+    [[ $output != *"tmpimg1"* ]]
+    [[ $output != *"tmpimg2"* ]]
+
+    # Delete list of images with invalid image
+    ch-image build -t tmpimg1 -f - . << 'EOF'
+FROM alpine:3.17
+EOF
+    ch-image build -t tmpimg2 -f - . << 'EOF'
+FROM alpine:3.17
+EOF
```

Comment:
Should encapsulate this somehow, e.g. in a function, rather than copy-pasting the code.